### PR TITLE
Add Major Changes page

### PR DESCRIPTION
**Pull request by travier**
Add new page to store documentation for major changes happening in
Fedora CoreOS.

Fixes https://github.com/coreos/fedora-coreos-docs/issues/314.

### CODE REVIEW

File: modules/ROOT/pages/major-changes.adoc [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+|`testing`| ...
+|`stable`| Will follow `testing` as usual
+|===
+
+=== Notes
+
+...
+////
+
+== Podman v4.0
```
Comment (cgwalters) [inline modules/ROOT/pages/major-changes.adoc:32]: Let's make this a hyperlink, or clearly link to https://fedoraproject.org/wiki/Changes/Podman4.0

**Comment by travier** [comment]
For https://github.com/coreos/fedora-coreos-tracker/issues/1106 & https://github.com/coreos/fedora-coreos-tracker/issues/676

**Review by cgwalters — APPROVED** [review]
This is looking good to me so far, thank you so much for doing this!

**Review by dustymabe — APPROVED** [review]
Looks good to me. Thank you @travier

**Comment by bgilbert** [comment]
Fixes https://github.com/coreos/fedora-coreos-docs/issues/314.

File: modules/ROOT/pages/major-changes.adoc [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+repo_gpgcheck=0
+enabled=1
+enabled_metadata=1
+$ sudo rpm-ostree override replace --experimental podman containers-common catatonit --freeze --from repo=copr:copr.fedorainfracloud.org:rhcontainerbot:podman4 --install aardvark-dns --install netavark
+$ sudo systemctl reboot
+----
+
+== Moving to iptables-nft
+
+=== Planing
```
Comment (jlebon) [inline modules/ROOT/pages/major-changes.adoc:83]:
```suggestion
=== Planning
```

File: modules/ROOT/pages/major-changes.adoc [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+|===
+
+=== Notes
+
+...
+////
+
+== Podman v4.0
+
+=== Planning
```
Comment (jlebon) [inline modules/ROOT/pages/major-changes.adoc:34]: Maybe before each `Planning` heading, we can have a small one sentence summarizing the change? E.g. for this the first sentence from the `Notes` section works (And then we can remove it from there). It feels funny to start with dates without first introducing it.

WDYT?

File: modules/ROOT/pages/major-changes.adoc [hunk 4]
```diff
@@ -0,0 +1,144 @@
+= Major Changes in Fedora CoreOS
+:toc:
+
+This is a list of major changes that were introduced in Fedora CoreOS with the
+notes associated with them.
+Those changes are also announced on the https://lists.fedoraproject.org/archives/list/coreos-status@lists.fedoraproject.org/[coreos-status mailing list].
+Note that this is not an exhaustive list of changes in Fedora CoreOS and only includes major changes that may require manual actions.
+This list is in reverse chronological order to keep recent changes at the top.
+
+////
```
Comment (jlebon) [inline modules/ROOT/pages/major-changes.adoc:10]: Minor: looks like the GitHub syntax renderer is confused by this and everything after the template isn't highlighted correctly. Not sure if there's a way to fix that. I guess we could put the template at the bottom of the doc instead?

**Review by jlebon — APPROVED** [review]
Some comments, but LGTM overall! Thanks for writing this!

File: modules/ROOT/pages/major-changes.adoc (hunk 2 above)
Comment (bgilbert) [inline modules/ROOT/pages/major-changes.adoc:83]: Planning

**Review by bgilbert — APPROVED** [review]
Looks great, thanks!

**Comment by travier** [comment]
Thanks for the reviews. Last update should resolve all comments.

**Review by jlebon — APPROVED** [review]
Looks great!

**Pull request merged**